result/flow.yaml

```diff
@@ -45,47 +47,31 @@ components:
               frames_rx_rate:
                 x-field-uid: 7
           x-field-uid: 3
-    Flow.MetricGroup.Request:
+    Flow.MetricTag.Filter:
```

Comment:
Does changing Flow.MetricGroup.Request to Flow.MetricTag.Filter  not break backwards compatibility?